sitback-furniture/src/page/home/home.jsx

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import CategoryCard from '../../components/cards/category-card/category-card';
+import styles from './home.module.scss';
+import Loader from '../../components/loader/loader';
+import { homeData } from '../../constants/string-constant';
+
+const Home = ({page})=>{
+
+    const [categoryData,setData] = useState();
+    const [isLoading,settIsLoading] = useState(true);
+
+    useEffect(()=>{
+        const fetchData = async ()=>{
+            
+            try{
+                const res = await axios.get('https://jsonmockserver.vercel.app/api/shopping/furniture/categories');
```

Comment:
Can you move this into a separate service file?

Reply:
api data retrieval function has bee moved to separate js service file